mne/channels/montage.py

```diff
@@ -348,6 +348,7 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         selection = selection[sel]
     else:
         ch_names_ = list(ch_names_)
+    ch_names_ = [str(ch) for ch in ch_names_]  # convert names to str
```

Comment:
I think it would be cleaner to set ``tolist()`` in places we are using ``np.str``.